src/AssetsPage/AssetsTable/index.jsx

```diff
@@ -38,6 +79,41 @@ class AssetsTable extends React.Component {
     }
   }
 
+  onSortClick(columnKey) {
+    // const prevColumnKey = this.state.sortState.columnKey;
+    const columns = [...this.state.tableColumns];
+    const column = this.state.tableColumns.find(col => (col.key === columnKey));
+    let newDirection;
+
+    if (this.state.sortState.columnKey === columnKey) {
+      newDirection = this.state.sortState.direction === 'descending' ? 'ascending' : 'descending';
+      column.sortDirection = newDirection;
+
+      // then we want to make the appropriate API call
+    } else {
+      // is this the best way to do a line break?
+      const oldColumn = this.state.tableColumns.find(col =>
+        (col.key === this.state.sortState.columnKey));
+
+      // can we have a better way to do this that's less ambiguous than relying on the default?
+      oldColumn.sortDirection = '';
```

Comment:
This is referring to the switch statement in getSortIcon; I'd rather there be a case for 'other' or 'none' or something, but it would be identical to the default case, I think. Unless the default case is ''?

Reply:
I'm okay with explicitly specifying '' as the default case - the 3 icons choices (`fa-sort`, `fa-sort-asc`, and `fa-sort-desc`) seem to correspond with those options well.

Reply:
I think 'none' or something that isn't empty string is more clear when reading the code (say if there's someone who hasn't seen this discussion) and it could either be left to fall into the default or be part of the switch right before the default so it still falls through to the default anyway.  It's a bit more to write/be cognizant of when writing for the future however to keep things unified.  I tend to lean towards readability, but am open to be swayed :)